src/elrail.cpp

```diff
@@ -124,6 +124,8 @@ static TrackBits GetRailTrackBitsUniversal(TileIndex t, byte *override)
  */
 static TrackBits MaskWireBits(TileIndex t, TrackBits tracks)
 {
+	if (likely(HasAtMostOneBit(tracks))) return tracks;
```

Comment:
Maybe add a comment like this?
```suggestion
	/* Single track bits are never masked out. */
	if (likely(HasAtMostOneBit(tracks))) return tracks;
```

Otherwise I think this is a sensible change.